components/developer/dtracetoolkit/manifests/sample-manifest.p5m

```diff
@@ -273,7 +273,9 @@ file path=opt/DTT/Code/Perl/hello_strict.pl
 file path=opt/DTT/Code/Perl/hello_strict.pl.~1~
 file path=opt/DTT/Code/Php/func_abc.php
 file path=opt/DTT/Code/Python/func_abc.py
+file path=opt/DTT/Code/Python/func_abc.py.~1~
```

Comment:
This should not be installed. Can we make sure it is not landing on the End user computers?

Reply:
The patch backup files all have a suffix of  "~1~".  A "gmake install" installs them to the prototype area, but a "gmake" publish ignores them.  This has always been the case and is still the case.  So, these files never land on  a user's desktop.  Your concerns are unfounded.

Reply:
Ah, good!